src/TimeGutter.jsx

```diff
@@ -1,39 +1,49 @@
-import React from 'react';
-import cn from 'classnames';
+import React, { Component, PropTypes } from 'react'
 import dates from './utils/dates';
-import localizer from './localizer'
+import TimeSliceGroup from './TimeSliceGroup.jsx'
 
-let TimeGutter = React.createClass({
+export default class TimeGutter extends Component {
```

Comment:
I think now that this is used for all the large slots, `TimeGutter` is no longer a good name for this component. Perhaps TimeSegment, or TimeColumn (to go with TimeGrid?) makes more sense.